Entity/File.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Vich\UploaderBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Embeddable
+ */
+class File
+{
+    /**
+     * @ORM\Column(name="name", nullable=true)
+     */
+    protected $name;
+
+    /**
+     * @ORM\Column(name="original_name", nullable=true)
+     */
+    protected $originalName;
+
+    /**
+     * @ORM\Column(name="mime_type", nullable=true)
+     */
+    protected $mimeType;
+
+    /**
+     * @ORM\Column(name="size", type="integer", nullable=true)
+     */
+    protected $size;
+
+    public function getName()
```

Comment:
I can add phpdocs with param/return types, but I haven't see any profit from method descriptions like `Sets the original file name`. It clear from method name what it do :)